tool/src/ec.rs

```diff
@@ -31,6 +31,8 @@ enum Cmd {
     LedSetValue = 12,
     LedGetColor = 13,
     LedSetColor = 14,
+    LedGetMode = 15,
```

Comment:
It is good to reserve these in the src/common/include/common/command.h file as well